.github/workflows/pr.yml

```
@@ -0,0 +1,100 @@
+  prepare:
+    name: "Prepare: Bootstrap the repository"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: npm install
+
+      # Cache node modules
+      - name: Cache Node Modules
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)